apps/src/code-studio/pd/fit_weekend1819_registration/FitWeekend1819Registration.jsx

```diff
@@ -38,12 +37,17 @@ export default class FitWeekend1819Registration extends FormController {
    * @override
    */
   getPageComponents() {
-    return [
-      Joining,
-      TravelPlans,
-      Releases,
-      Confirmation
-    ];
+    if (this.state.data.ableToAttend === 'Yes') {
```

Comment:
this seems redundant with the early `shouldShowSubmit`; is there a specific reason for this?

Reply:
So the bottom looks like this

![image](https://user-images.githubusercontent.com/14246419/36281813-e35c9fd0-1253-11e8-891f-687a47102b0d.png)

Reply:
Ah, just to avoid showing the page changer?

If that's a general concern, we might want to consider updating the base `FormController` to be smarter about showing the page changer when `shouldShowSubmit` indicates an early submit, but that's an issue for a future PR